buildHarness/buildfuzz.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Question, where did this file come from? Did you author yourself or is this coopted from upstream? Is there a reason make did not work here for fuzzing (I see you've added afl, but this is not required for fuzzing)?

Reply:
I author myself, it's the build script from upstream but just changed Line 16 to build with afl-clang, that way it'll build the binary examples for afl.